lms/djangoapps/ytp/management/commands/update_user_fullname.py

```diff
@@ -0,0 +1,73 @@
+"""
+Script for update User's first name, last name  and the name field in UserProfile
+"""
+import json
+from logging import getLogger
+
+import requests
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models import Q
+from social_django.models import UserSocialAuth
+
+log = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Update a User's first name, last name  and the name field in UserProfile from remote service .
+    """
+    help = "Update user's fullname"
+ 
+    def handle(self, *args, **options):
+       """
+       Realisation the command to update User's first name, last name  and the name field in UserProfile.
+       """
+       self.stdout.write("Start to update User's first name and last name  and the name field in UserProfile")
+       user_social_auth_list = UserSocialAuth.objects.filter(
+           Q(user__first_name__icontains='') | Q(user__last_name__icontains='') | Q(user__profile__name__icontains='')
```

Comment:
Do we need `icontains` for filtering empty strings? It seems  user__first_name='' is fair enough or I miss something?

Reply:
You are right. user__first_name='' id fair enough. I fixed it.